erpnext/controllers/print_settings.py

```diff
@@ -1,9 +1,28 @@
 # Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
 # License: GNU General Public License v3. See license.txt
 
+from __future__ import unicode_literals
+import frappe
+from frappe.utils import cint
+
 def print_settings_for_item_table(doc):
+	
 	doc.print_templates = {
 		"description": "templates/print_formats/includes/item_table_description.html",
 		"qty": "templates/print_formats/includes/item_table_qty.html"
 	}
+	customised_print_preview = cint(frappe.db.get_value("Features Setup", None, "customised_print_preview"))
```

Comment:
Please use 'compact_item_view` here. The field names and labels should be clear to the end user.